app/rot-carrier/app.toml

```diff
@@ -34,7 +34,7 @@ priority = 3
 max-sizes = {flash = 16384, ram = 4096}
 stacksize = 2048
 start = true
-uses = ["rom", "secure_syscon", "syscon", "flash"]
+uses = ["rom", "secure_syscon", "syscon", "flash", "update_details"]
```

Comment:
It's unclear why we don't get alignment errors on read here. Probably just luck. This should be changed from a `uses` section to a memory array using `MaybeUninit` similar to https://github.com/oxidecomputer/hubris/blob/d9cf6c70d5cb505797e4d78e150db318ec463c8c/task/net/src/buf.rs#L17-L19

Reply:
Fixed.